pipeline/runtime/run.go

```diff
@@ -41,6 +41,9 @@ func executeRunStep(ctx context.Context, engine *engine.Engine, r *api.StartStep
 		step.Command[0] += getOutputVarCmd(step.Entrypoint, r.OutputVars, outputFile)
 	}
 
+	artifactFile := fmt.Sprintf("%s/%s-artifact", pipeline.SharedVolPath, step.ID)
+	step.Envs["PLUGIN_ARTIFACT_FILE"] = artifactFile
```

Comment:
I will suggest using DRONE_ARTIFACT_FILE instead of PLUGIN_ARTIFACT_FILE

Reply:
For k8 we use PLUGIN_ARTIFACT_FILE right ? Customers can have their plugins which might be generating artifacts and expects PLUGIN_ARTIFACT_FILE as env variable ?